packages/honeycomb-opentelemetry-web/src/web-vitals-autoinstrumentation.ts

```diff
@@ -419,7 +423,7 @@ export class WebVitalsInstrumentation extends InstrumentationAbstract {
 
   onReportCLS = (
     cls: CLSMetricWithAttribution,
-    applyCustomAttributes?: ApplyCustomAttributesFn,
+    applyCustomAttributes: ApplyCustomAttributesFn = noop,
```

Comment:
Do we want the default to be a noop here? If that's the case, we'll always call this function on every report and that feels like a step further than checking an if statement. I would rather have it as an optional and do the branch check to avoid the function call everytime but if there is some context I'm missing let me know!

Reply:
Backed out here 51a94f7, we can revisit in the future :awesome-cat: